styles/styles.css

```
@@ -79,6 +80,36 @@ html {
   text-decoration: none;
 }
 
+.navigation-link:hover {
+  color: #756257;
+}
+
+.navigation-link:hover.navigation-icon {
+  fill: #756257;
+}
+
+.navigation-link:focus {
+  color: #756257;
+  border: 3px solid #83b3d3;
+  border-radius: 10px;
+}
+
+.navigation-link:focus.navigation-icon {
+  color: #756257;
+  border: 3px solid #83b3d3;
+  border-radius: 10px;
+}
+
+.navigation-link:active {
+  color: #756257;
+  opacity: 0.3;
+}
+
+.navigation-link:active.navigation-icon {
```

Так ты выбираешь элемент, у которого есть оба класса.
 выбрать иконку внутри ссылки, нужно писать один класс не _вместе_ с другим, а _после_:
 ```suggestion
.navigation-link:active .navigation-icon {
```